core_backend/tests/api/test_question_answer.py

```diff
@@ -619,6 +628,11 @@ async def async_fake_generate_speech(*args: Any, **kwargs: Any) -> str:
             assert "error" in json_response
             assert response.status_code == 500
```

Comment:
You don't need to do that again since you checked for that here:
` assert response.status_code == expected_status_code`